_STAGE2_JSPACE/experiments/exp_010b_small/RESULTS_EXP010B.md

```
@@ -0,0 +1,174 @@
+(tensor basins = greedy leader clustering of terminal mean vectors at the
+0.999 gate threshold; via-tail = one pass through layers j+1..11, decode at
+11):
+
+| arm | window | conv | locks | tensor basins (sizes) | decode terminals | via-tail agree |
+|---|---|---|---|---|---|---|
+| SB | 0→11 | 14/25 | all 120 | 7 (6,5,4,3,3,2,2) | ` Divine`×11, ` prolet`×10, ` till`×3, ` Anarch`×1 | 25/25 (tail empty) |
+| S1 | 0→5 | 25/25 | all 120 | 1 (25) | ` the`×25 | 25/25 |
+| S2 | 3→8 | 25/25 | all 120 | 25 (all singletons) | 21 unique: fragments/punct (`er`×3, ` ,`×2, `ur`×2, `lex`, ` just`, ` how`, ` Animated`, ` )`, `'`, `:`, `pan`, NBSP-run, `,`, `o`, ` Sky`, ` the`, `ison`, `y`, `-`, `]`, `Yes`) | **2/25** |
+| S3 | 6→11 | 25/25 | all 120 | 5 (13,5,4,2,1) | `.`×25 | 25/25 (tail empty) |
+| S4 | 0→8 | 25/25 | all 120 | 1 (25) | ` the`×25 | 25/25 |
+| S5 | 3→11 | **1/25** | D02 @180 | 7 (11,5,5,1,1,1,1) | `<|endoftext|>`×23, `.`×1 (G01, capped), ` to`×1 (D02, converged) | 25/25 (tail empty) |
```

markdownlint-cli2 (0.23.0)

[warning] Spaces inside code span elements

(MD038, no-space-in-code)